lua/wikis/dota2/Links/CustomData.lua

```diff
@@ -13,5 +13,9 @@ return {
 			player = 'https://www.faceit.com/en/players/',
 			match = 'https://www.faceit.com/en/dota2/room/',
 		},
+		blasttv = {
```

Comment:
Won't this effect rocketleague? 
https://blast.tv/rl/tournaments/birmingham-major-1-2025
https://liquipedia.net/rocketleague/Rocket_League_Championship_Series/2025/Birmingham_Major

Reply:
No, this file is dota2-specific